src/store/modules/AccessControl/LocalUserMangementStore.js

```diff
@@ -19,19 +19,42 @@ const LocalUserManagementStore = {
     getUsers({ commit }) {
       api
         .get("/redfish/v1/AccountService/Accounts")
-        .then(response => {
-          return response.data.Members.map(user => user["@odata.id"]);
-        })
-        .then(userIds => {
-          return api.all(userIds.map(user => api.get(user)));
-        })
+        .then(response => response.data.Members.map(user => user["@odata.id"]))
+        .then(userIds => api.all(userIds.map(user => api.get(user))))
         .then(users => {
           const userData = users.map(user => user.data);
           commit("setUsers", userData);
         })
-        .catch(error => {
-          console.log(error);
-        });
+        .catch(error => console.log(error));
+    },
+    createUser(_, { username, password, privilege, status }) {
+      const data = {
+        UserName: username,
+        Password: password,
+        RoleId: privilege,
+        Enabled: status
+      };
+      api
+        .post("/redfish/v1/AccountService/Accounts", data)
+        .then(() => this.dispatch("localUsers/getUsers"))
+        .catch(error => console.log(error));
+    },
+    updateUser(_, { originalUsername, username, password, privilege, status }) {
+      const data = {};
+      if (username) data.UserName = username;
+      if (password) data.Password = password;
+      if (privilege) data.RoleId = privilege;
+      if (status !== undefined) data.Enabled = status;
+      api
+        .patch(`/redfish/v1/AccountService/Accounts/${originalUsername}`, data)
```

Comment:
Awesome stuff! 

This function could also be used to change the password on expired password on login? 
Any thoughts on how we would share it in this case?

Reply:
Would be the same as what we have in Angular right? Would just need to reference this store module in the login component when password reset is required

Reply:
Sounds good.